_docs/glider-dac-communications-plan.md

```diff
@@ -19,7 +19,7 @@ Currently we have the following URLs that contain some piece of the glider story
 # Technical Docs and Github Repos
```

Comment:
This whole page will be gone when merge https://github.com/ioos/glider-dac/pull/261. I would not bother reviewing it.